Adhesion/Interactions/ES_Coulomb.py

```diff
@@ -36,61 +36,67 @@
 
 
 class ES_C(Potential):
-    """ Electrostatic potential for Coulombic interaction when a voltage V is applied across an air gap between two perfect insulators of thicknesses d1,d2 and permittivities eps1,eps2
-
+    """ Electrostatic potential for Coulombic interaction when a voltage is applied across a gap between two perfect insulators
+@doi https://doi.org/10.3389/fmech.2020.567386
     """
 
     name = "es-c"
 
-    def __init__(self, eps1, eps2, d1, d2, V, communicator=MPI.COMM_WORLD):
+    def __init__(self, eps0, eps1, epsg = 1, eps2, d1, d2, voltage, communicator=MPI.COMM_WORLD):
```

Comment:
Name changed to voltage
eps0 is now an input
eps1 is now an input (set to 1 as default, for air gap)